app/controllers/admin/organization/users_controller.rb

```diff
@@ -0,0 +1,29 @@
+class Admin::Organization::UsersController < ApplicationController
+
+  def index
+    @organization = Organization.find_by(slug: params[:organization_slug])
+    @admins = @organization.users
+    # require "pry"; binding.pry
+  end
+
+  def new
+    @organization = Organization.find_by(slug: params[:organization_slug])
+    @user = @organization.users.new
+  end
+
+  def create
+    organization = Organization.find_by(slug: params[:organization_slug])
+    role = Role.find_or_create_by(name: "org_admin")
+    user = User.find_by(username: params[:user][:username])
+    if user
+      user.roles << role
+      org_users = organization.users
+      org_users << user
+      # flash[:success] = "You've added #{user.username} as an admin"
```

Comment:
@jmejia 
It seems that the user is not saving at this point. I put a byebug in and when I tried updating the user's organization with a bang, I get the error that validation failed because password could not be blank. So it successfully finds the user, but can't update any of its relationships.

(byebug) user.update!(organization_id: organization.id)
   (0.2ms)  BEGIN
  User Exists (0.3ms)  SELECT  1 AS one FROM "users" WHERE ("users"."username" = 'angela@example.com' AND "users"."id" != 1) LIMIT 1
   (0.2ms)  ROLLBACK
*** ActiveRecord::RecordInvalid Exception: Validation failed: Password can't be blank

nil